lib/lambdaws.js

```diff
@@ -31,18 +35,26 @@ var versions = {
 var relativeDir = path.dirname(module.parent.parent.filename);
 
 function _resolvePathFromParent(p) {
+    
+    var prependingStr = '';
+    if(p.startsWith(global.constants.INTERNALS_PREPENDING_STR)) {
```

Comment:
startsWith makes things clear but is it useful ? the simplest way of doing this is ``` p[0] === your_tested_string ``` anyways, I don't mind it's the startsWith code can't produce any bugs since it's dead simple